crates/cxx-qt-lib/src/gui/qfont.rs

```diff
@@ -368,8 +368,8 @@ pub use ffi::{
 
 #[repr(C)]
 pub struct QFont {
-    _cspec: MaybeUninit<i32>,
-    _resolve_mask: MaybeUninit<u16>,
+    _cspec: MaybeUninit<usize>,
+    _resolve_mask: MaybeUninit<i32>,
```

Comment:
small thing but shouldn't this be `u32` not `i32` as it's std::uint32 / uint on the C++ side ?

Reply:
good catch